proposals/description/tip_121b.ts

```diff
@@ -0,0 +1,180 @@
+import { TemplatedProposalDescription } from '@custom-types/types';
+
+// FEI balance on the TC timelock
+const TC_FEI_BALANCE = '2733169815107120096987175';
+
+// TRIBE balance on the TC timelock
+const TC_TRIBE_BALANCE = '2733170474316903966022879';
+
+const tip_121b: TemplatedProposalDescription = {
+  title: 'TIP_121b: Protocol ops and technical cleanup',
+  commands: [
+    // 1. Withdraw LUSD and DAI from LUSD->DAI LBP swapper, send to LUSD holding deposit
+    {
+      target: 'lusdToDaiSwapper',
+      values: '0',
+      method: 'exitPool(address)',
+      arguments: (addresses) => [addresses.daiHoldingPCVDeposit],
+      description: 'Move all DAI and LUSD to daiHoldingPCVDeposit'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.daiHoldingPCVDeposit, // pcvDeposit
+        addresses.lusd, // token
+        addresses.lusdHoldingPCVDeposit, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: `Move all LUSD from daiHoldingPCVDeposit to lusdHoldingPCVDeposit.`
+    },
+    // 2. Withdraw WETH and DAI from WETH->DAI LBP swapper, move to Lido
+    {
+      target: 'ethToDaiLBPSwapper',
+      values: '0',
+      method: 'exitPool(address)',
+      arguments: (addresses) => [addresses.daiHoldingPCVDeposit],
+      description: 'Move all DAI and WETH to daiHoldingPCVDeposit'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.daiHoldingPCVDeposit, // pcvDeposit
+        addresses.weth, // token
+        addresses.aaveEthPCVDeposit, // to
```

Comment:
Transfer to the `wethHoldingPCVDeposit` and then pull from there to the Lido Deposit